### ISSUE TITLE
Project Settings: Implement project-setting-provided default values

### ISSUE THREAD

**Issue by tjcouch-sil** [paranext/paranext-core#710]
As an extension developer, I want to be able to get project settings' values including defaults so I can stay in sync whether or not the setting has been set.

----

On the project settings service:
- Maintain list of all project settings' default values (hard-coded; we will load from extensions in #721)
  - Include some real settings - books present, language, versification
- Implement `papi.projectSettings.getDefault(key)` - provides the default value from the manifest instead of the parameter default value passed in. No more `defaultValue` parameter in service or in hook
  - If project setting is provided in extensions or built into core (though for now the list is just built into core as mentioned above), returns setting default
  - If setting is not provided anywhere, throws
    - Should we revert this and say `defaultValue` parameter can be a backup for the extension-provided default? I think it would be nice never to have to worry about if a setting exists
- Double check to make sure the notes on the PSI MandatoryDataTypes `Settings` data type mentions `getSetting` should call `papi.projectSettings.getDefault(key)` if they don't have a value for the setting

Depends on #419, #509

**Comment by tjcouch-sil** [comment]
Closed - merged into #509 as this is trivially more work, and it would be a waste of time to create the other api then remove it to add this one.
